.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - "2.3"
+  - "2.3.0"
   - "jruby"
```

Comment:
Should we make this list consistent with the others? The other had jruby9.1, 1.7.23, 2.5 etc.